server/service.go

```diff
@@ -162,6 +163,14 @@ func CAKeyPassword(pw []byte) ServiceOption {
 	}
 }
 
+// optional argument to set the validity of signed client certs in days
+func ClientValidity(duration int) ServiceOption {
+	return func(s *service) error {
+		s.clientValidity = duration
```

Comment:
isnt 
`flClDuration = flag.String("crtvalid", envString("SCEP_CERT_VALID", "365"), "validity for new client certificates in days")` setting the default?